Matlab_Scripts/5_Depth_Colourless/Study/Analysis_VR_P3_Depth_Colourless.m

```diff
@@ -0,0 +1,166 @@
+% %% load in datasets
+function Analysis_VR_P3_Depth_Colourless(exp,anal)
```

Comment:
if this function takes these inputs, than master.m should reflect that , how do you call this when you run the analysis and preprocessing, put those calls in master.m, so that the analysis is completely linear and reproducable